bsp/imxrt/libraries/drivers/drv_sai.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * File: drv_sound.h
+ *
+ * COPYRIGHT (C) 2012-2018, Shanghai Real-Thread Technology Co., Ltd
+ */
+
+#ifndef __DRV_SOUND_H_
+#define __DRV_SOUND_H_
+
+#include <rtthread.h>
+#include <rtdevice.h>
+#include <board.h>
+#include <rtdef.h>
+
+#include <rthw.h>
+#include "fsl_sai.h"
+#include "fsl_dmamux.h"
+#include "fsl_sai_edma.h"
+
+#define AUD_DMA_FIFO_SIZE (2048)
+#define	CODEC_I2C_NAME  ("i2c1")
```

Comment:
Please take care of indentation.

Reply:
The modified document has been submitted.